Libplanet/Net/Messages/Message.cs

```diff
@@ -72,7 +72,7 @@ internal enum MessageType : byte
             /// A reply to <see cref="GetRecentStates"/>.
             /// Contains the calculated recent states and state references.
             /// </summary>
-            RecentStates = 0x0f,
+            RecentStates = 0x13,
```

Comment:
I guess `0x0g` is safer than `0x13` because `0x13` can be used for another message type in the older version.

Reply:
@longfin Do you mean `0x14`?

Reply:
I tried it but realized that the hexadecimal number is up to `f`. :cry:

Reply:
Ah, I've confused. nevermind.